api.yaml

```diff
@@ -322,7 +322,65 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
-
+  /v1/cloud/arnid/{arnid}:
```

Comment:
do we need resource specific to arnid? can we have path /v1/cloud/{arnid} instead?

Reply:
From my understanding, yes because there are other endpoints that find resources by IP address and hostname. This path can be used to find resources specific to an ARN ID.